### PR TITLE
{devel}[GCCcore/6.4.0] Qt5 5.10.1 + deps

### PR DESCRIPTION
**Pull request by geimer**
Factored out from #6328

### CODE REVIEW

**Comment by geimer** [comment]
Test report by @geimer
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
swatc01 - Linux debian 9.3, Intel(R) Xeon(R) CPU           X5660  @ 2.80GHz, Python 2.7.13
See https://gist.github.com/10b8c6ad25e0c46172687c05071a1114 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (5 easyconfigs in this PR)
node2574.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/ba1288e3ba320d47cfec15633aa63aa3 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
node2065.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/1632eef04af6575e18c02862e546a8f9 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 5 out of 6 (5 easyconfigs in this PR)
easybuild.novalocal - Linux centos linux 7.5.1804, Intel Xeon E312xx (Sandy Bridge), Python 2.7.5
See https://gist.github.com/b0d2ab2677b963b8235a43710348b8bf for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (5 easyconfigs in this PR)
easybuild.novalocal - Linux centos linux 7.5.1804, Intel Xeon E312xx (Sandy Bridge), Python 2.7.5
See https://gist.github.com/eb7644f907b7a993d917fa27efd541cb for a full test report.

**Comment by boegel** [comment]
@verdurin broken test report was because `/dev/shm` was used?

**Comment by verdurin** [comment]
@boegel yes, I have a little menagerie of nodes across the UK for these tests, not all of which have lots of RAM...

File: easybuild/easyconfigs/m/Mako/Mako-1.0.7-GCCcore-6.4.0-Python-2.7.14-bare.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    '4e02fde57bd4abb5ec400181e4c314f56ac3e49ba4fb8b0d50bba18cb27d25ae',  # Mako-1.0.7.tar.gz
+]
+
+builddependencies = [
+    # use same binutils version that was used when building GCCcore
+    ('binutils', '2.28'),
+]
+
+dependencies = [
+    ('Python', '2.7.14', '-bare'),
```
Comment (boegel) [inline easybuild/easyconfigs/m/Mako/Mako-1.0.7-GCCcore-6.4.0-Python-2.7.14-bare.eb]: Before merging this, I'd like to settle the discussion on https://github.com/easybuilders/easybuild-easyconfigs/pull/5072

**Comment by boegel** [comment]
Thanks for clarifying @verdurin

File: easybuild/easyconfigs/s/setuptools/setuptools-39.1.0-GCCcore-6.4.0-Python-2.7.14-bare.eb [hunk 2]
```diff
@@ -0,0 +1,33 @@
+easyblock = 'PythonPackage'
+
+name = 'setuptools'
+version = '39.1.0'
+versionsuffix = '-Python-%(pyver)s-bare'
+
+homepage = 'http://pypi.python.org/pypi/setuptools/'
```
Comment (verdurin) [inline easybuild/easyconfigs/s/setuptools/setuptools-39.1.0-GCCcore-6.4.0-Python-2.7.14-bare.eb]: @geimer this should be reverted to `https`, unless there's a good reason for the change.

**Comment by akesandgren** [comment]
All but setuptools are codes that probably would benefit quite a lot from being compiled by Intel instead of GCC. So from my point of view they should be moved up a step in the toolchain. I.e., to GCC (or higher) instead of GCCcore, and then have an intel version of them too.

And Qt5 5.10.1 needs, according to https://github.com/easybuilders/easybuild-easyconfigs/pull/7086, a gperf dependency.

**Comment by geimer** [comment]
With #8201 being merged, this PR is now obsolete.

**Comment by boegel** [comment]
Thanks for following up on this and closing the PR @geimer!

**Pull request closed without merging**